ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/NotifierSelection.tsx

```diff
@@ -68,7 +68,9 @@ function NotifierSelection({
     function onNotifierChange(_id, selectionId) {
         const notifierObject = notifiers.find((notifier) => notifier.id === selectionId);
         if (notifierObject) {
+            const notifierMailingLists = notifierObject.labelDefault.split(',');
             setFieldValue('notifier', notifierObject);
+            setFieldValue('mailingLists', notifierMailingLists);
```

Comment:
This works, but I can foresee a user typing in a custom list, realizing they selected the wrong notifier, and then wondering where their custom list went when they can notifiers.

@kcarmichael08 @gaurav-nelson If this is the behavior that makes it into the final feature, we should explicit call it out in the docs.

Reply:
That's a valid consideration. I have two thoughts about this:

1. We unlink the notifier selection and the distribution list after the distribution list has some user-generated value. We would need to keep track of whether the value in the input was due to the default value provided by the notifier selection or if the user actually modified it. It might also be nice to have some sort of reset button to reset to the default value provided by the notifier selection.
2. We introduce a confirmation prompt. When a user inputs a custom list, and selects a new notifier, then we show them a confirmation prompt asking them if they'd like to reset the list to the default one provided by the notifier or keep their custom list. This could be a bit interruption though.

Reply:
We don't have to decide for this PR, since we are up against a workflow checkpoint.

Let's just ask in the next weekly sync.

Reply:
@zhenpesky can you remind me what we decided on this?

Reply:
@sachaudh we didn't have a chance to discuss this in the weekly sync. Let's bring it up in the next sync then.
The idea from the mock is a little tricky because we have to connect the 'reset button' concept with the placeholder text and use it as a default value of a distribution list (clear all custom values would show the default distribution list which is also shown in a format of placeholder text). There is a better proposal here, let me know what you all think:

Scenario: 
Step 1: The user selects a notifier (i.e. notifier X) that is not what the user wants.
(system: show the default distribution list)

Step 2: The user enters a custom value in the distribution list. 
(system: show the [‘undo' icon + button link “reset to default”])

Step 3: The user realizes the wrong notifier was chosen and then switches to the desired notifier (i.e. notifier A).
(system: After switching is done, DO NOT change the custom distribution list)

Step 4: The user wants to use the default distribution list.
(system: Keep the custom distribution list until Step 5 is taken)

Step 5: The user clicks [‘undo' icon + button link “reset to default”] OR removes all custom values in the distribution list)
(system: Show the default distribution list based on the selected filter; remove the [‘undo' icon + button link “reset to default”]) 

Question: For the undo icon and button link: do we usually disable it instead of remove or hide? To keep the consistent behavior, the same behavior would expected to happen in the ‘none’ option in the ‘day’ selection dropdown under the ‘Schedule’ section.

Reply:
@zhenpesky This looks good to me. Can you provide a mock just so I know what this should look like? I can also send you a screenshot of what I did so you can let me know if that's good with you

Reply:
@sachaudh the mock is provided [here](https://issues.redhat.com/browse/ROX-18726) in the comment section.